### ISSUE TITLE
Auto sort the cards in a hand

### ISSUE THREAD

**Issue by lennertcl** [lennertcl/durak#15]
Add a function that automatically sorts the cards in your hand. Should be called every time cards are added or removed.

Cards should probably be sorted based on number, but trump cards should be kept together all the way to the right of the hand.

**Comment by ihsin** [comment]
Hi @lennertcl , I want to work on this. Can you assign me?

**Comment by lennertcl** [comment]
Here you go

**Comment by ihsin** [comment]
Tried running the game but  I can't see the cards. I think the /static/images directory is missing from repo.

**Comment by lennertcl** [comment]
Yes it's in the .gitignore but it actually probably shouldn't be so I will change this.

**Comment by ihsin** [comment]
Thanks! I can see the cards now.

**Comment by ihsin** [comment]
Btw... I created a Dockerfile for it. If you have any plans to add then I can create a pull request.

**Comment by lennertcl** [comment]
Yeah you can pull request if you want. I will add it if it doesn't break other things.
